manticore/core/workspace.py

```diff
@@ -507,15 +512,6 @@ def save_summary(self, state, message):
             summary.write(f"Command line:\n  '{' '.join(sys.argv)}'\n")
             summary.write(f'Status:\n  {message}\n\n')
 
-            # FIXME(mark) This is a temporary hack for EVM. We need to sufficiently
-            # abstract the below code to work on many platforms, not just Linux. Then
-            # we can remove this hack.
-            if getattr(state.platform, 'procs', None) is None:
-                import pprint
-                summary.write("EVM World:\n")
-                summary.write(pprint.pformat(state.platform._global_storage))
```

Comment:
This is some old code that doesn't work in the end. There is no `_global_storage` or `global_storage` anywhere in the codebase (and so it crashed with a missing `AttributeError`).